NDVALPHA-UPDATED-LOCAL-DEVICE-REDIRECTION (#94631)
Azure/azure-content: articles/virtual-desktop/configure-device-redirections.md

### USB redirection

>[!IMPORTANT]

> >To redirect to the remote Session Host a mass storage USB device connected to the local computer you still need to configure the **Drive/storage redirection** RDP Property, simply enabling the *USB redirection* won't work. Refer to the *Local drive redirection* below to learn how to manage those.

Reviewer: ```suggestion
>To redirect a mass storage USB device connected to your local computer to the remote session host, you'll need to configure the **Drive/storage redirection** RDP property. Enabling the **USB redirection** RDP property by itself won't work. For more information, see [Local drive redirection](#local-drive-redirection).
```
